app/src/main/java/com/github/se/cyrcle/model/parking/ParkingViewModel.kt

```diff
@@ -70,8 +70,14 @@ class ParkingViewModel(
    * @param parking the parking to add
    */
   fun addParking(parking: Parking) {
+    val tile = Tile.getTileFromPoint(parking.location.center)
+    tilesToParking.value[tile] = tilesToParking.value[tile]?.plus(parking) ?: listOf(parking)
```

Comment:
With regards to what Maria said, we still add it to the tiles even on failliure?

Reply:
Oops forgot to remove the old implementation, thankfully you checked my work